data-sets-zowe-server-package/src/main/resources/bin/configure.sh

```diff
@@ -1,77 +0,0 @@
-#!/bin/sh
-
-################################################################################
-# This program and the accompanying materials are made available under the terms of the
-# Eclipse Public License v2.0 which accompanies this distribution, and is available at
-# https://www.eclipse.org/legal/epl-v20.html
-#
-# SPDX-License-Identifier: EPL-2.0
-#
-# Copyright IBM Corporation 2019, 2020
-################################################################################
-
-# Source main utils script
-. ${ROOT_DIR}/bin/utils/utils.sh
-ensure_java_is_on_path
```

Comment:
Sorry, something I noticed after reviewing the APIML components - this check was kept in configure there. Is there a reason for the different behaviour in data-sets and jobs?

Reply:
Yes this is in prepare-environments https://github.com/zowe/zowe-install-packaging/blob/staging/bin/internal/prepare-environment.sh#L142. I'm kind mixed due to the fact of we may want to make java/node optional.... I think I should also delete configure.sh on api-layer.

Reply:
Updated api-layer https://github.com/zowe/api-layer/pull/1002/commits/6f073ac532349640a2daf706e62c922702d17fc3